mlir/include/mlir/Conversion/MathToLLVM/MathToLLVM.h

```diff
@@ -23,6 +24,7 @@ class Pass;
 
 void populateMathToLLVMConversionPatterns(const LLVMTypeConverter &converter,
                                           RewritePatternSet &patterns,
+					  PatternBenefit benefit = 1,
                                           bool approximateLog1p = true);
```

Comment:
nit: I would prefer if you made PatternBenefit the last argument. Just prevents churn and generally PatternBenefit is the last argumenet in populate functions.